README.md

```diff
@@ -106,44 +91,77 @@ Starting local OpenShift cluster...
 
    To login as administrator:
        oc login -u system:admin
+```
+Note that, the IP seen above is dynamic and can change. It can be retrieved with `minishift ip`.
 
-# Adding 'oc' to the PATH (Note, concrete version might be different)
+ **Note:** By default Minishift uses the driver most relevant to the host OS.
+ To use a driver of choice for Minishift, use the `--vm-driver=xxx` flag with `minishift start`. For example, to use VirtualBox instead of KVM for Fedora, use: `$ minishift start --vm-driver=virtualbox`.
+
+4. Add `oc` binary to the
+_PATH_ (Note, concrete version might be different):
+ ```sh
 $ export PATH=$PATH:~/.minishift/cache/oc/v1.3.1
+```
+5. Login to your OpenShift account and authenticate yourself:
 
-# Authenticate against OpenShift
+ ```sh
 $ oc login https://192.168.99.128:8443 -u developer -p developer
+```
+
+<a name="usage"></a>
+## Usage
 
-# Create Node.js example app
+This section contains a brief demo of Minishift usage. For details on the usage of Minishift refer to the [Using Minishift](/docs/using.md) guide.
```

Comment:
"This section contains a brief demo of using OpenShift as provisioned by Minishift"

Really, the demo is not about Minishift, but rather OpenShift.